### ISSUE TITLE
Section list columns too wordy - plus section description not searchable

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#876]
The column headings in the section panel list are needlessly long. I'll look into making them more concise.

Also, the section description field in not available as a search criteria. Might want to see about fixing that.

**Comment by Bloke** [comment]
Missing section search by description was an oversight. Fixed in 426ec482, thanks.

**Comment by Bloke** [comment]
Regarding the wording, how about:

- On default page?
- Syndicate articles?
- Searchable?

Or is that too terse?

**Comment by philwareham** [comment]
Terse is good, yes I'll amend in RPC to that soon. Thanks.

**Comment by cara-tm** [comment]
Am I missing something? Is there a "Description" column into the main sections list? Why do you want to search into a non displaying column? Otherwise, that is very useful.

**Comment by phiw13** [comment]
@cara-tm There is description on the Edit panel, but it does not appear on the list panel, and searching for a string that is contained in the description field returns the section owning that description.

+1 on the shorter column headings. Much better for narrower displays

**Comment by philwareham** [comment]
@cara-tm there are other lists where you can search criteria that isn't in a column. It's still useful.

**Comment by philwareham** [comment]
Done.

**Comment by cara-tm** [comment]
@phiw13 My question was: why didn't you add a "Descriprion" choice, uncked by default (so you preserve shorter columns headings), into the (new) list options feature?
